pylint-2.7

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env vpython3
+# Copyright 2019 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+# Note: We use tags here instead of refs because refs will always hit the
+# network to see if there are any updates.
+# [VPYTHON:BEGIN]
+# # Third party dependencies.  These are only listed because pylint itself needs
+# # them.  Feel free to add/remove anything here.
+# python_version: "3.8"
+#
+# wheel: <
+#   name: "infra/python/wheels/futures-py2_py3"
```

Comment:
This relative path and the ones below don't exist in an engine checkout. Is this expected to work on developers' machines?